extensions/spring-web/resteasy-reactive/runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -1,5 +1,5 @@
 ---
 artifact: ${project.groupId}:${project.artifactId}:${project.version}
-name: "Spring Web RESTEasy Reactive"
+name: "Spring Web Quarkus REST"
```

Comment:
Maybe keep this one `Spring Web REST`?

Reply:
BTW, I'm not entirely sure what would be best here.

Reply:
Yeah, I agree `Spring Web Quarkus REST` looks like a buzzword salad. I've looked at the names of the other Spring ones, and apart from "Spring Web RESTEasy Classic," all the others use a different pattern, so there's not much to learn. It's unlisted, so it also doesn't matter too much - but I'll drop the 'Quarkus'. It's situations like this that 'qrest' or the other distinctive novelty names would have been ideal for. :)